meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -45,7 +69,11 @@ do_install:append() {
         fi
     else
         cat ${WORKDIR}/domu-vdevices.cfg >> ${CFG_FILE}
-        cat ${WORKDIR}/pvr-${XT_DOMU_CONFIG_NAME} >> ${CFG_FILE}
+        if ${@bb.utils.contains('MACHINE_FEATURES', 'gsx', 'true', 'false', d)}; then
+            cat ${WORKDIR}/gsx-common.cfg >> ${CFG_FILE}
+            sed -i "s/cma=32M/cma=256M pvrsrvkm.DriverMode=1/g" ${CFG_FILE}
```

Comment:
We had many issues with `sed` earlier. I'd prefer some other solution. If you already building part of config dynamically in this recipe, you may build commandline as well.